src/NLog/LayoutRenderers/CounterLayoutRenderer.cs

```diff
@@ -82,6 +82,18 @@ public CounterLayoutRenderer()
         /// <param name="builder">The <see cref="StringBuilder"/> to append the rendered data to.</param>
         /// <param name="logEvent">Logging event.</param>
         protected override void Append(StringBuilder builder, LogEventInfo logEvent)
+        {
+            int v = GetNextValue(logEvent);
+
+            builder.AppendInvariant(v);
+        }
+
+        /// <summary>
+        /// Get raw value, updates the sequence
+        /// </summary>
+        object IRawValue.GetRawValue(LogEventInfo logEvent) => GetNextValue(logEvent);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'logEvent' to 'logEventInfo' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS927)